### DISCUSSION TITLE
host, basePath and scheme in the OpenAPI description

### DISCUSSION THREAD

**Discussion by anthonny** [supabase/supabase#3906] [Questions]
Hello 👋,

I have a question about the OpenAPI description generated by PostgREST.

Here is a screen of the description that I have:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/2006548/141841149-9260171f-0fd3-49bc-8440-4d07f3f84331.png">

As you can see, the `host` is set to `0.0.0.0:3000`, the`basePath` is `/` and the `scheme` is `http`.
So we I use tools like `openapi-gen`, these values are hard coded in the generated code (I'm looking for a way to use env vars 😊).

Maybe I'm wrong, but I suppose that the values should be `host: xxxxx.supabase.co`, `basePath: /rest/v1` and `scheme: https`.
What do you think about this point? 
Tell me if I'm totally wrong (maybe I missed something) or if there is already an issue about this point 🙏.

I really like your product 
😘😘😘

**Comment by chasers** [comment]
Hi Anthony!

You're right there doesn't seem to be an issue for this that I can find. Can you create one please? Are you using a workaround currently? If so, can you describe it in the issue also? I'll get some eyes on this but until then I'm sure your workaround will be helpful for other people.

**Comment by chasers** [comment]
I added [this](https://github.com/supabase/supabase/issues/3913) if you want to expand on it.
